assertions/src/main/java/io/camunda/zeebe/process/test/inspections/DeploymentInspections.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2021 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.camunda.zeebe.process.test.inspections;
+
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.process.test.filters.DeploymentRecordStreamFilter;
+import io.camunda.zeebe.protocol.record.value.deployment.DeploymentResource;
+import java.io.ByteArrayInputStream;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.assertj.core.api.Assertions;
+import org.camunda.bpm.model.xml.instance.DomElement;
+
+/**
+ * Inspections for deployment events. This is useful to find technical identifiers for
+ * human-readable element names.
+ */
+public class DeploymentInspections {
+  private final DeploymentRecordStreamFilter deploymentRecordStreamFilter;
+
+  public DeploymentInspections(final DeploymentRecordStreamFilter deploymentRecordStreamFilter) {
+    this.deploymentRecordStreamFilter = deploymentRecordStreamFilter;
+  }
+
+  /**
+   * Finds the BPMN element id by its name.
+   *
+   * <p>Keeps the test human-readable when asserting on bpmn elements. Asserts that there is only
+   * one element for the given name to prevent mistakes.
+   *
+   * @param elementName the name of the BPMN element
+   * @return the id of the found BPMN element
+   */
+  public String findBpmnElementId(String elementName) {
+    return findElementId(deploymentRecordStreamFilter.getBpmnDeploymentResources(), elementName);
+  }
+
+  /**
+   * Finds the BPMN element id by its name inside the referenced BPMN process.
+   *
+   * <p>Keeps the test human-readable when asserting on bpmn elements. Asserts that there is only
+   * one element for the given name to prevent mistakes.
+   *
+   * @param bpmnProcessId the id of the deployed process
+   * @param elementName the name of the BPMN element
+   * @return the id of the found BPMN in the given process
+   */
+  public String findBpmnElementId(String bpmnProcessId, String elementName) {
```

Comment:
```suggestion
  public String getBpmnElementId(String bpmnProcessId, String elementName) {
```

🔧 Generally find is used when something returns an `Optional`, or in the earlier days the `elementId` or `null`. As this method will fail if it doesn't exist it's cleaner to change this to `get`. Please adjust the javadoc as well 🙂

Reply:
Fixed in https://github.com/camunda/zeebe-process-test/pull/813/commits/741b41591b16536bd8db80f64ab703c69e6d321b